### DISCUSSION TITLE
Ocorreu uma exceção: CLR/System.Data.SqlClient.SqlException

### DISCUSSION THREAD

**Discussion by franciscoguerra** [balta-io/forum#26] [Tira Dúvidas] [answered]
Carreira backend
Acesso à dados com .NET, C#, Dapper e SQL Server 
AULA-> CRUD
Deu erro no método public static void CreateUser() 
os metodos ReadUsers(); ReadUser(); deu certo mais CreateUser() estou dificuldade. de já agradeço.
creio que tem haver com a coluna no bando de dados, porém não consegui resolver.
![1](https://user-images.githubusercontent.com/12381369/192159504-e868cf56-667d-462d-b8ed-545966a4492e.png)
![2](https://user-images.githubusercontent.com/12381369/192159506-505212d6-b1bd-4441-8b39-37947e165d19.png)
![3](https://user-images.githubusercontent.com/12381369/192159507-c22c0a93-0910-4d43-a1df-16bdd7c0a857.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @franciscoguerra, como vai?

Na sua tabela está `Password` e no C# está `PasswordHash`.
